frontend/src/app/features/team-planner/team-planner/planner/team-planner.component.ts

```diff
@@ -206,7 +207,11 @@ export class TeamPlannerComponent extends UntilDestroyedMixin implements OnInit,
     this.principalIds$,
     this.showAddAssignee$,
   ]).pipe(
-    map(([principals, showAddAssignee]) => !principals.length && !showAddAssignee),
+    debounceTime(250),
```

Comment:
if all you need to debounce is the table stop, perhaps it's better to put that in it's separate observable or `tap` instead of delaying the full `isEmpty$` state

Reply:
I assume that this would not solve the original problem because the empty template would already be shown and 250ms later the loading indicator vanishes. Depending on when it actually starts, the user will see it again over the template. So I fear, we have to delay the whole observable.